### ISSUE TITLE
[joss][sec] possible injection and data leakage

### ISSUE THREAD

**Issue by anibalsolon** [neuroanatomy/thresholdmann#39]
Part of: https://github.com/openjournals/joss-reviews/issues/6336

Loading a malicious JSON file can execute code and leak MRI data (or worse). A simple validation for numeric values during loading can prevent such scenario.

```json
{"points":[[100,164,"0\"><script>window.alert('hey jude')</script><br style=\"display: none;\" data-close=\""]],"values":[10]}
```

**Comment by katjaq** [comment]
Thank you for the note. We hadn't thought about this as usually the user themselves is the producer of the json file they'll later load. We'll integrate a sanitizer.

**Comment by katjaq** [comment]
A check for numbers only has been implemented in # [6bdaf8c](https://github.com/neuroanatomy/thresholdmann/commit/6bdaf8cb84d9b4f00423263ff56400630a5b361f). Thank you @r03ert0 .
